packages/mobile/index.js

```diff
@@ -2,6 +2,7 @@ import 'react-native-gesture-handler'
 import { AppRegistry, LogBox } from 'react-native'
 
 import { name as appName } from './app.json'
+import 'text-encoding-polyfill'
```

Comment:
Should add a comment why we need this imo

Reply:
totes